src/utils/cache/getWallet.ts

```diff
@@ -127,6 +90,78 @@ export const getWallet = async <TWallet extends EVMWallet>({
       wallet = await getLocalWallet({ chainId, walletAddress });
       break;
 
+    case WalletType.smartAwsKms: {
+      if (accountAddress)
+        throw new Error(
+          "Smart backend wallet cannot be used to operate external smart account",
+        );
+
+      const splitArn = splitAwsKmsArn(walletDetails.awsKmsArn);
+
+      const personalWallet = new AwsKmsWallet({
```

Comment:
Same nit: this is the adminWallet or signerWallet (pick one), so keep naming consistent.